project/deps.sc

```diff
@@ -1,7 +1,7 @@
 import mill._, scalalib._
 
 object Versions {
-  def ammonite      = "2.4.0-14-4824b429"
+  def ammonite      = "2.5.0"
```

Comment:
Ammonite 2.5.x depends on PPrint 0.7.x
```
> coursier resolve com.lihaoyi:::ammonite-compiler:2.5.0
...
com.lihaoyi:pprint_2.13:0.7.1:default
```
but mdoc depends on 0.6.x
https://github.com/scalameta/mdoc/blob/v2.2.24/build.sbt#L123

I think we need to stick to older version of Ammonite that contains a compatible version of PPrint: `2.4.1`
https://github.com/com-lihaoyi/Ammonite/blob/2.4.1/build.sc#L96

Would you mind creating a ticket over at mdoc to upgrade to newer version of PPrint? (Could you also, please, mention me?)

Reply:
I opened an issue over at doc. I also had a look at it and concluded that I would get far with attempting that change myelf, sadly. As requested, you got an  @ in the issue. 

Thanks again for your help.